app/containers/MessageBox/index.js

```diff
@@ -823,9 +849,152 @@ class MessageBox extends Component {
 		}
 	}
 
+	changeComposerState = () => {
```

Comment:
maybe this would be better names `toggleFullScreen`. The current name isn't clear what it is actually doing

Reply:
Good idea thanks!